qiskit/transpiler/passes/optimization/litinski_transformation.py

```diff
@@ -40,28 +41,28 @@ def __init__(self, fix_clifford=True):
         """LitinskiTransformation initializer.
 
```

Comment:
We should skip these, they look weird in the IBM docs render 🙂 
```suggestion
        """
```

Reply:
sure [dd461cc](https://github.com/Qiskit/qiskit/pull/14753/commits/dd461ccf25d11f4f070e0d7d351b68d5e116fb86)